pom.xml

```diff
@@ -86,7 +106,9 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Export-Package>hu.blackbelt.osgi.liquibase.*;version=${project.version},liquibase;version=${liquibase.version},liquibase.*;version=${liquibase.version}</Export-Package>
+                        <Export-Package>
+                            hu.blackbelt.osgi.liquibase.*;version=${project.version},liquibase;version=${liquibase.version},liquibase.*;version=${liquibase.version},org.yaml.*;version=${snakeyaml.version}
```

Comment:
org yaml  (snakeyaml) cannot be exported, because it have to be private.

Reply:
Removed.